docs/user-guide/aes-operator.md

```diff
@@ -0,0 +1,141 @@
+    * the final GA 4.0.
+    * any incremental and bug fix versions 4.* and 4
+    * the most recent 4.* EA release (i.e., if 4.0.5 is the last GA version and
+      there is a 4.1-EA3, then this selects 4.1-EA3 over the 4.0.5 GA).
+
+Read more about SemVer [here](https://github.com/Masterminds/semver#basic-comparisons).
+
+`updateWindow` is an optional item that will control when the updates can take
+place. This is used to force system updates to happen late at night if that’s
+what the sysadmins want.
+
+There can be any number of `updateWindow` entries (separated by commas).
+Never turns off automatic updates even if there are other entries in the
+comma-separated list. Never is used by sysadmins to disable all updates during
+blackout periods by doing a `kubectl` apply or using our Edge Policy Console to
+set this.
+
+Each `updateWindow` is in crontab format (see https://crontab.guru/) Some
+examples of `updateWindow` are:
+
+* `0-6 * * * SUN`: every Sunday, from 0am to 6am
+* `5 1 * * *`: every first day of the month, at 5am
+
+The Operator cannot guarantee minute time granularity, so specifying a minute in the crontab expression can lead to some updates happening sooner/later than expected.
+
+## Install the AES Operator
+
+You can install the AES Operator in a number of ways:
+
+* Manually
+* Helm Chart
+* OperatorHub.io
+
+### Install Manually
+
+Start by installing the operator:
+
+1. Create the Operator Custom Resource schema with the following command: `kubectl apply -f https://github.com/datawire/ambassador-operator/releases/download/latest/ambassador-operator-crds.yaml`
+2. Install the actual CRD for the Ambassador Operator in the `ambassador` namespace with the following command: `kubectl apply -n ambassador -f https://github.com/datawire/ambassador-operator/releases/download/latest/ambassador-operator.yaml`
+3. To install the Ambassador Operator CRD in a different namespace, you can specify it in `NS` and then run the following command:
+
+```shell
+$ NS="custom-namespace"
+$ curl -L https://github.com/datawire/ambassador-operator/releases/download/latest/ambassador-operator.yaml | \
+    sed -e "s/namespace: ambassador/namespace: $NS/g" | \
+    kubectl apply -n $NS -f -
+```
+
+Then, create the `AmbassadorInstallation` Custom Resource schema and apply it to the AES Operator.
+
+1. To create the `AmbassadorInstallation` Custom Resource schema, use the following YAML as your guideline: [https://github.com/datawire/ambassador-operator#the-operator-custom-resource-cr](https://github.com/datawire/ambassador-operator#the-operator-custom-resource-cr).
+2. Save that file as `amb-install.yaml`
+3. Edit the `amb-install.yaml` and optionally complete configurations such as Version constraint or UpdateWindow:
+4. Finally, apply your `AmbassadorInstallation` CRD to the AES Operator schema with the following command: `kubectl apply -n ambassador -f amb-install.yaml`
+
+### Install via OperatorHub.io
```

Comment:
need to remove this section as this is scheduled for next cycle

Reply:
removed